### DISCUSSION TITLE
How to zoom particular part of GLTF model on single tap in Android?

### DISCUSSION THREAD

**Discussion by amit-iottive** [google/filament#7865] [Q&A]
Hi,
I have used gltf model. I want to zoom clicked part. how can I achieve this?
I have attached model image for reference.
Thanks

![Screenshot_20240517_155220](https://github.com/google/filament/assets/89839738/de6595cc-89e1-4012-9e10-62cd67b26d41)

**Comment by darshan-bhadeshiya-1098** [comment]
I also need the same solution. @amit-iottive have you found any solution?
